html-forms/index.html

```diff
@@ -7,23 +7,26 @@
     <h1>Coding Form</h1>
     <form>
       <label for="email">Email</label> <br />
-      <input type="text" name="email" size="20"/> <br />
-      <label for="coding is cool">Why is coding cool?</label><br />
-      <select name="Coding">
+      <input type="email" name="email" size="20"/> <br />
+      <label for="coding-is-cool">Why is coding cool?</label><br />
+      <select name="Coding" id= "Coding-is-cool">
```

Comment:
Whoops! The "C" in "coding" is capitalized here, but not in your `for` attribute on the label. These will not actually match up.